tests/test_core.py

```diff
@@ -200,6 +201,17 @@ def test_request():
     assert R._filename_zip[0] == get_remote_file("images/chelsea.zip")
     assert R.filename == get_remote_file("images/chelsea.zip") + "/chelsea.png"
 
+    # Test zip false-positive detection
+    with tempfile.TemporaryDirectory() as tmp_path:
+        bait_zip = Path(tmp_path) / "test.zip"
+        bait_zip.mkdir()
+        file_path = bait_zip / "foo.jpg"
+        chelsia = get_remote_file("images/chelsea.png")
+        shutil.copy(chelsia, file_path)
```

Comment:
The Py3.5 CI build fails on `shutil.copy` with `PosixPath` objects. Can you please convert to string here? :)

Reply:
Sure. I migrated to old good os.path.

But now there is a weird problem with CI...